inst/safetyGraphics_app/server.R

```diff
@@ -52,13 +52,15 @@ observeEvent(settings_new$status(),{
 
 
 # set up all chart tabs from the start (all_charts defined in global.R)
-  for (chart in all_charts){
+  for (row in 1:nrow(chartsMetadata)){
+    chart<-chartsMetadata[row,"chart"]
+    chartLabel<-chartsMetadata[row,"label"]
     tabid <- paste0(chart, "_tab_title")
 
     appendTab(
       inputId = "nav_id",
       tab = tabPanel(
-        title = chart,
+        title = chartLabel,
```

Comment:
I believe we need to add ```value = chart``` after the title = chartLabel so that the hide/show tab part works

Reply:
agreed - I added that in the `widget-update` PR